tests/e2e/specs/word-ads.test.js

```diff
@@ -1,20 +1,15 @@
-/**
- * WordPress dependencies
- */
-import { createNewPost } from '@wordpress/e2e-test-utils/build/create-new-post';
 /**
  * Internal dependencies
  */
 import BlockEditorPage from '../lib/pages/wp-admin/block-editor';
 import PostFrontendPage from '../lib/pages/postFrontend';
 import WordAdsBlock from '../lib/blocks/word-ads';
 import { connectThroughWPAdminIfNeeded } from '../lib/flows/jetpack-connect';
-import { execShellCommand } from '../lib/utils-helper';
+import { execShellCommand, resetWordpressInstall, getNgrokSiteUrl } from '../lib/utils-helper';
 
 // Activate WordAds module if in CI
 async function activateWordAdsModule() {
-	let cmd =
-		'docker-compose -f ./tests/e2e/bin/docker-compose.yml run --rm -u 33 cli_e2e_tests wp jetpack module activate wordads';
+	let cmd = './tests/e2e/docker/whatever.sh cli "wp jetpack module activate wordads"';
```

Comment:
I am a bit confused by the `whatever.sh` here. :)

Reply:
:D oops. This is a part of the docker e2e setup, which is not merged yet. It used only for local runs.